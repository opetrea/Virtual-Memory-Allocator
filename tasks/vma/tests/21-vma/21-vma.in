ALLOC_ARENA 65536
ALLOC_BLOCK 4096 10
ALLOC_BLOCK 12288 10
ALLOC_BLOCK 12308 10
PMAP
ALLOC_BLOCK 12298 10
PMAP
WRITE 4096 26 Observ sisteme de operare
READ 4096 14
FREE_BLOCK 12298
WRITE 12308 24 Sper sÄƒ nu iau SEGFAULT
PMAP
DEALLOC_ARENA
