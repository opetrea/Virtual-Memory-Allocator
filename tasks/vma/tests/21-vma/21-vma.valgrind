==91907== Memcheck, a memory error detector
==91907== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==91907== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==91907== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==91907== Parent PID: 91906
==91907== 
==91907== 
==91907== HEAP SUMMARY:
==91907==     in use at exit: 0 bytes in 0 blocks
==91907==   total heap usage: 40 allocs, 40 frees, 9,154 bytes allocated
==91907== 
==91907== All heap blocks were freed -- no leaks are possible
==91907== 
==91907== For lists of detected and suppressed errors, rerun with: -s
==91907== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
