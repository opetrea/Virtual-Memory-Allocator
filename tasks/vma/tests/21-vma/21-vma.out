Total memory: 0x10000 bytes
Free memory: 0xFFE2 bytes
Number of allocated blocks: 3
Number of allocated miniblocks: 3

Block 1 begin
Zone: 0x1000 - 0x100A
Miniblock 1:		0x1000		-		0x100A		| RW-
Block 1 end

Block 2 begin
Zone: 0x3000 - 0x300A
Miniblock 1:		0x3000		-		0x300A		| RW-
Block 2 end

Block 3 begin
Zone: 0x3014 - 0x301E
Miniblock 1:		0x3014		-		0x301E		| RW-
Block 3 end
Total memory: 0x10000 bytes
Free memory: 0xFFD8 bytes
Number of allocated blocks: 2
Number of allocated miniblocks: 4

Block 1 begin
Zone: 0x1000 - 0x100A
Miniblock 1:		0x1000		-		0x100A		| RW-
Block 1 end

Block 2 begin
Zone: 0x3000 - 0x301E
Miniblock 1:		0x3000		-		0x300A		| RW-
Miniblock 2:		0x300A		-		0x3014		| RW-
Miniblock 3:		0x3014		-		0x301E		| RW-
Block 2 end
Warning: size was bigger than the block size. Writing 10 characters.
Warning: size was bigger than the block size. Reading 10 characters.
Observ sis
Warning: size was bigger than the block size. Writing 10 characters.
Total memory: 0x10000 bytes
Free memory: 0xFFE2 bytes
Number of allocated blocks: 3
Number of allocated miniblocks: 3

Block 1 begin
Zone: 0x1000 - 0x100A
Miniblock 1:		0x1000		-		0x100A		| RW-
Block 1 end

Block 2 begin
Zone: 0x3000 - 0x300A
Miniblock 1:		0x3000		-		0x300A		| RW-
Block 2 end

Block 3 begin
Zone: 0x3014 - 0x301E
Miniblock 1:		0x3014		-		0x301E		| RW-
Block 3 end
